py-shared/src/quilt_shared/pkgpush.py

```diff
@@ -72,8 +72,8 @@ class S3CopyLambdaParams(pydantic.BaseModel):
 
 
 class ChecksumType(str, enum.Enum):
-    MP = "QuiltMultipartSHA256"
-    SP = "SHA256"
+    MODERN = "sha2-256-chunked"
+    LEGACY = "SHA256"
```

Comment:
I'd say we should use `SHA256` and `SHA256_CHUNKED`

Reply:
i'm not opposed to that in principle, but you probably should have said that earlier, since we've already painted all our bike sheds this color

Reply:
well, I've said that I don't like "modern" at the very beginning and we agreed that chunked is good especially if we'll decide to always double hash

Reply:
do you propose to use these ids on the python level? e.g.
```suggestion
    SHA256_CHUNKED = "sha2-256-chunked"
    SHA256 = "SHA256"
```

Reply:
or for both enum variants and the actual strings?

Reply:
> do you propose to use these ids on the python level? e.g.

yes